content/millennium/r4/financial/coverage.md

```diff
@@ -147,6 +147,40 @@ X-Runtime: 3.890282
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+### Example - Encounter-level Coverage
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Coverage
+
+### Body
+
+<%= json(:r4_coverage_encounter_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Tue, 22 Oct 2019 15:59:33 GMT
+Etag: W/"0"
+Last-Modified: Tue, 22 Oct 2019 15:59:30 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Coverage/31363178-11500257
+Server-Response-Time: 3890.363996
+Status: 201 Created
```

Comment:
Let's remove the status header.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/357/commits/ec65adaf23ee1ac3262d7653afda097101a198f4#diff-81b4004b2ae7fd9a288412597630810cL138